README.md

```diff
@@ -10,11 +10,16 @@ Go clients for talking to a [kubernetes](http://kubernetes.io/) cluster.
 * The `transport` package is used to set up auth and start a connection.
 * The `tools/cache` package is useful for writing controllers.
 
-### Releases
+### Versions
```

Comment:
For easy reference, redirectors are defined [here](https://github.com/kubernetes/k8s.io/blob/339bb00db924a43798b33bd6dfc2aa8e514ca538/k8s.io/configmap-www-golang.yaml#L69-L70).